nsinit/nsenter.go

```diff
@@ -1,84 +0,0 @@
-package main
-
-import (
-	"fmt"
-	"log"
-	"net"
-	"os"
-	"strconv"
-	"strings"
-	"text/tabwriter"
-
-	"github.com/docker/libcontainer/configs"
-	"github.com/docker/libcontainer/devices"
-	"github.com/docker/libcontainer/mount/nodes"
-	"github.com/docker/libcontainer/namespaces"
-	_ "github.com/docker/libcontainer/namespaces/nsenter"
-)
-
-// nsenterExec exec's a process inside an existing container
-func nsenterExec(config *configs.Config, args []string) {
-	if err := namespaces.FinalizeSetns(config, args); err != nil {
-		log.Fatalf("failed to nsenter: %s", err)
-	}
-}
-
-// nsenterMknod runs mknod inside an existing container
-//
-// mknod <path> <type> <major> <minor>
-func nsenterMknod(config *configs.Config, args []string) {
```

Comment:
We have talked about needing to add devices and things like that that may not entail running a binary inside. We don't have too many real usecases so can punt for now.